cellprofiler/gui/figure.py

```diff
@@ -1918,6 +1935,9 @@ def subplot_scatter(self, x, y,
         axes.set_xscale(xscale)
         axes.set_yscale(yscale)
 
+        if self.dimensions == 3:
```

Comment:
We should probably just do this in the future and avoid `matplotlib.gridspec` altogether!

Reply:
Wait the `gridspec` is pretty neat haha, I'd say we move everything over to it!